src/tcgwars/logic/impl/gen4/DiamondPearlPromos.groovy

```diff
@@ -629,6 +630,13 @@ public enum DiamondPearlPromos implements LogicCardInfo {
           pokePower "Leap Away", {
             text "Once during your turn , if Toxicroak is your Active Pok�mon, you may flip a coin. If heads, return Toxicroak and all cards attached to it to your hand. This power can�t be used if Toxicroak is affected by a Special Condition."
             actionA {
+              assert self.active : "$self is not your Active Pokémon"
+              checkLastTurn()
+              checkNoSPC()
+              powerUsed()
```

Comment:
Minor suggestion, but since implementing: Would maybe add a confirm when it's the only pokemon in play before using the power, similar to Super Scoop Up.

Might be worth its own PR though, maybe adding a static assert for all cards like this, that checks for no bench and in that case does the confirm.

Reply:
I'd prefer just getting these to work "good enough" for testing purposes. Primarily just don't want to have to review literally every Gen 4 PR just in case it needs to use scoop up.

Reply:
The rest of these should all be part of said scoop-up check PR. The existing ones, and the ones I added, avoid the issue by just not allowing the move be used. I don't think it's closely related enough to this PR to handle here.